doc/client_lua_api.txt

```diff
@@ -709,7 +709,12 @@ Call these functions only at load time!
     * `spec` is a `SimpleSoundSpec`
     * `parameters` is a sound parameter table
 * `minetest.sound_stop(handle)`
-
+### Key Input
+* `minetest.register_on_keydown(func(key))`
+    * Called when player presses a key.
+    * If the player holds a key down, it'll be called multiple times.
```

Comment:
Suggestion: `Is called accordingly to the keyboard character repetition interval when the key is kept pressed.`
Also needs Irrlicht reference (see below)